src/components/macro/macro-action-editor/tab/macro-mouse/macro-mouse.component.ts

```diff
@@ -0,0 +1,95 @@
+import {
+    Component,
+    OnInit,
+    Input,
+    ViewChild
+} from '@angular/core';
+
+import {Tab} from '../../../../popover/tab/tab';
+import {MacroAction, macroActionType} from '../../../../../config-serializer/config-items/MacroAction';
+
+import { PressMouseButtonsMacroAction } from '../../../../../config-serializer/config-items/PressMouseButtonsMacroAction';
+import { HoldMouseButtonsMacroAction } from '../../../../../config-serializer/config-items/HoldMouseButtonsMacroAction';
+import { ReleaseMouseButtonsMacroAction } from '../../../../../config-serializer/config-items/ReleaseMouseButtonsMacroAction';
+
+enum TabName {
+    Move,
+    Scroll,
+    Click,
+    Hold,
+    Release
+}
+
+@Component({
+    selector: 'macro-mouse-tab',
+    template: require('./macro-mouse.component.html'),
+    styles: [
+        require('../../macro-action-editor.component.scss'),
+        require('./macro-mouse.component.scss')
+    ],
+    host: { 'class': 'macro__mouse' }
+})
+export class MacroMouseTabComponent implements OnInit {
+    @Input() macroAction: MacroAction;
+    @ViewChild('tab') selectedTab: Tab;
+
+    private activeTab: TabName;
+    private tabName = TabName;
```

Comment:
Change it to:
```
/* tslint:disable:variable-name: It is an enum type. So it can start with uppercase. */
/* tslint:disable:no-unused-variable: It is used in the template. */
private TabName = TabName;
/* tslint:enable:no-unused-variable tslint:enable:variable-name */
```